showcases/screen-reader/tests/navigation.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { getTest, testDefault } from '../default';
+
+const test = getTest();
+
+test.describe('DBNavigation', () => {
+	testDefault({
+		test,
+		title: 'default',
+		description:
+			'should have texts inline or as data-label attributes (next())',
+		url: './#/05/navigation?page=density',
+		async testFn(voiceOver, nvda) {
+			if (nvda) {
+				await nvda?.next(); // Navigation Item 1
+				await nvda?.act(); // Open menu
```

Comment:
```suggestion
				await nvda?.next(); // Navigation Item 2
				await nvda?.previous(); // Navigation Item 1
				await nvda?.act(); // Open menu
```

even though that NVDA and VoiceOver wouldn't be feature equal in this case, we would check whether the navigation is actually hidden from NVDA screenreader in the first place before interacting with the button.

Reply:
Why do we need this? NVDA tells us with `..., menu button, collapsed,...` that the sub-navigation is hidden isn't it?

Reply:
It tells us that we've set an attribute that would announce `collapsed`, but it doesn't test for whether the navigation is actually visually hidden.